wasi-tools/wit/deps.toml

```diff
@@ -0,0 +1,4 @@
+io = "https://github.com/WebAssembly/wasi-io/archive/v0.2.0.tar.gz"
```

Comment:
The wit files are fetched using the [wit-deps](https://github.com/bytecodealliance/wit-deps) tool.
Everything in wasi-tools/wit/deps/* is generated by it.